packages/commuter-client/src/reducers/index.js

```diff
@@ -6,8 +6,20 @@ const commuter = (state = {}, action) => {
     case types.REQUEST_CONTENTS:
       return Object.assign({}, state, { isFetching: action.isFetching });
     case types.RECEIVE_CONTENTS:
+    // filter out hidden files
+      let content;
+      switch (action.entry.type) {
+        case 'directory':
+          content = action.entry.content.filter((file) => {
+            return !file.name.startsWith('.');
+          });
+        break;
+      default:
+        return action.entry.content;
```

Comment:
I have the fix in #152. I'll go ahead and extract my changes there so I can finish up local mode right after fixing master.